src/gui/helpers/nav_helper.rs

```diff
@@ -82,18 +86,20 @@ impl GuiHelper for NavHelper {
         };
         ui.separator();
 
-        egui::CollapsingHeader::new("Gameobject Coordinates NOT IMPLEMENTED")
+        egui::CollapsingHeader::new("Gameobject Coordinates")
             .default_open(true)
             .show(ui, |ui| {
-                let position = managers.player_party_manager.data.position;
+                let position = managers.player_party_manager.data.gameobject_position;
                 let pos_x = format!("{:.3}", position.get_x());
                 let pos_y = format!("{:.3}", position.get_y());
                 let pos_z = format!("{:.3}", position.get_z());
                 ui.label(format!("x: {}", pos_x));
                 ui.label(format!("y: {}", pos_y));
                 ui.label(format!("z: {}", pos_z));
 
-                if ui.button("Copy to clipboard").clicked() {
+                if ui.button("Copy to clipboard NOT IMPLEMENTED").clicked() {
```

Comment:
Is it copy to or copy from?

Reply:
click and copy to clipboard

Reply:
Ah, I get what's going on then. The "copied text from the clipboard" below threw me off.

Reply:
SORRY